docs/topics/conditional-view-processing.txt

```diff
@@ -198,6 +201,14 @@ to compute the ETag and last modification values in all situations. In fact,
 you **should** use the same functions, so that the same values are returned
 every time.
 
+.. note::
+
+    Django does not automatically set the ``ETag`` and ``Last-Modified``
```

Comment:
The condition decorator only sets the ETag and Last-Modified headers for safe HTTP methods, i.e. GET and POST. (explain why?)